pkgs/development/compilers/scala-runners/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchFromGitHub, jre, coursier }:
+
+stdenv.mkDerivation rec {
+  pname = "scala-runners";
+  version = "unstable-2020-02-02";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "dwijnand";
+    rev = "95e03c9f9de0fe0ab61eeb6dea2a364f9d081d31";
+    sha256 = "0mvlc6fxsh5d6gsyak9n3g98g4r061n8pir37jpiqb7z00m9lfrx";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin $out/lib
+    sed -ie "s_\Wcs\W_ ${coursier}/bin/coursier _" scala-runner
```

Comment:
```suggestion
    sed -ie "s|\Wcs\W|${coursier}/bin/coursier|" scala-runner
```

Reply:
`|` are preferred over `_`?

Reply:
``|`` or ``/`` are prefered. ``_`` is really unusual for nixpkgs.